js/quadency.js

```diff
@@ -94,6 +95,10 @@ module.exports = class quadency extends Exchange {
                 'ACCESS-TIMESTAMP': strTs,
                 'QUADX': 'true',
             };
+            if (this.addData) {
+                const key = Object.keys (this.addData)[0];
```

Comment:
Why not iterate through the keys and set each value in the header? With this setup, addData is limited to only one key value pair which may be become an issue if we decided there is more that needs to be passed. Also maybe change the name to addHeaders or customHeaders? Makes it more explicit what this object is for.